_config.yml

```
@@ -56,15 +56,16 @@ defaults:
       type_of_guidance: "Test Rule"
       rule_meta:
         rule_type: "atomic rule"
-      sidebar:
-        footer:
-          content: Test Rules describe ways to test conformance to WCAG success criteria. They are <a href="../about">for evaluation</a>.
-      standalone_resource_nav_links:
-        - name: All Rules
-          ref: /standards-guidelines/act/rules/
-        - name: About Rules
+      sidebar: true
+      standalone_resource_header:
+        title: WCAG 2 Test Rules
+        subtitle: For developers of evaluation tools and test methodologies
```

```suggestion
        subtitle: For accessibility testers and developers of test tools and methodologies
```

I think the current wording is too specific. ACT rules are written for anyone doing testing.

---

@shawna-slh ?

---

Per previous comments, I think this deserves a separate issue. Done: https://github.com/w3c/wcag-act-rules/issues/93